api-get-policy.go

```diff
@@ -51,12 +52,18 @@ func (c Client) ListBucketPolicies(bucketName, objectPrefix string) (bucketPolic
 		return map[string]policy.BucketPolicy{}, err
 	}
 	policyInfo, err := c.getBucketPolicy(bucketName)
-	if err != nil {
+	errResponse := ToErrorResponse(err)
+	if err != nil && errResponse.Code != "NoSuchBucketPolicy" {
```

Comment:
I think it is better if we return map[string]policy.BucketPolicy{} when we have NoSuchBucketPolicy without going to policy.GetPolicies() later.